src/motor_control.c

```diff
@@ -38,6 +38,13 @@ void motor_control_configure(MotorControl *mc, const RefloatConfig *config) {
     mc->click_current = config->startup_click_current;
     mc->parking_brake_mode = config->parking_brake_mode;
     mc->main_freq = config->hertz / 2;
+
+    // set parking brake true/false here instead of each cycle
```

Comment:
The "... instead of each cycle" refers to the change you're making, not the semantics of the code. It's irrelevant outside the context of this PR, please remove the comment, it's only going to be confusing when encountered after the PR is merged.